cpp/src/round/round.cu

```diff
@@ -75,7 +75,13 @@ struct round_fn {
     cudaStream_t stream,
     rmm::mr::device_memory_resource* mr)
   {
-    auto result   = cudf::make_fixed_width_column(input.type(), input.size());
+    auto result = cudf::make_fixed_width_column(input.type(),
+                                                input.size(),
+                                                copy_bitmask(input, stream, mr),
+                                                cudf::UNKNOWN_NULL_COUNT,
```

Comment:
Can't you copy the null count from the input too?

Reply:
My bad, oversight. Fixed.